docs/framework/migration-guide/how-to-determine-which-versions-are-installed.md

```diff
@@ -98,6 +98,7 @@ Users can install and run multiple versions of the .NET Framework on their compu
     |On Windows 10 Anniversary Update: 394802<br /><br /> On all other OS versions: 394806|[!INCLUDE[net_v462](../../../includes/net-v462-md.md)]| 
     |On Windows 10 Creators Update: 460798<br/><br/> On all other OS versions: 460805 | .NET Framework 4.7 |
     |On Windows 10 Fall Creators Update: 461308<br/><br/> On all other OS versions: 461310 | .NET Framework 4.7.1 |
+    |On Windows 10 RS4:     <br/><br/> On all other OS versions:    | .NET Framework 4.7.2 |
```

Comment:
reminder to add the values here

Issue #3991 also asks whether all other OS versions includes other Windows 10 releases and also has some good feedback on how to make this more consistent and easier for folks creating scripts. Something perhaps for after the release?

Reply:
Yes, definitely, @mairaw. I've assigned it to myself. As a short term fix, I've added "only" to the Windows 10 descriptions.